CHANGES/3483.feature

```diff
@@ -0,0 +1 @@
+Internal Server Errors in plain text if the browser does not support HTML.
```

Comment:
No.
I'll run `towncrier` on making 3.5.2 release.
It will delete change files and update the root `CHANGES.rst`.
After that I'll merge 3.5 branch into master, it will delete `CHANGES/3483.feature` in master as well.